### DISCUSSION TITLE
Field options should not allow text which isn't in single or multiselect / Make 'tag' box uneditable

### DISCUSSION THREAD

**Discussion by aliya-ryan** [digidem/mapeo-desktop#578] [Ideas]
**Describe the idea**
_Testing MD 5.3.0-beta.2_
Currently if you create points in MD Territory, or edit elements introduced from Mm, the list of fields is editable. In the select fields you can only select items that are on the single or multiselect view - which seems logical. However the fields and content are also listed below, in the "All Tags" box, and here you can edit the tags and add text which doesn't appear as options in the fields - which means that you can effectively create new tags from those in the configuration. It feels like this would give way to possible errors. 
Maybe the All Tags box could be made ineditable, or even invisible - not sure if there is a reason for it to be displayed to our users.

**Comment by gmaclennan** [comment]
The "all tags" section can be useful if observations / points / lines / areas have additional properties which are not defined in the presets. This can happen when presets change over time, and perhaps do not have a field defined for all the tags defined for a particular thing on the map. In the Observation view the split is "Additional details", which essentially means "tags not defined in the preset" as opposed to "all tags". It would make sense if the map editing view mimicked this behaviour (e.g. "all tags" became "other tags").

On a related note, currently in Observation View for single/multi-select fields it is possible to enter items which are not in the list (although the UI nudges you towards selecting an existing option). This can be a "feature" if someone wants to add a new classification when it does not exist in the preset, but could also not be desirable in some situations.

We could add a configuration option for the presets, about whether an "other" option is available on single/multi-select fields, e.g. whether the user is able to enter something that is not in the pre-defined list of options.

Thoughts?

**Comment by okdistribute** [comment]
This sounds like a great idea. It's more confusing for the user to have too many options and can cause issues.

> The "all tags" section can be useful if observations / points / lines / areas have additional properties which are not defined in the presets. This can happen when presets change over time, and perhaps do not have a field defined for all the tags defined for a particular thing on the map. In the Observation view the split is "Additional details", which essentially means "tags not defined in the preset" as opposed to "all tags". It would make sense if the map editing view mimicked this behaviour (e.g. "all tags" became "other tags").
> 

'Additional details' in the Observation view of both Mobile and Desktop doesn't allow you to /add/ new tags, so it seems like maybe you should be able to /see/ tags that somehow were added before, but not /add new ones/ -- is that right?
